packages/codemirror/style/index.css

```diff
@@ -23,8 +23,6 @@
 
 .CodeMirror {
   line-height: var(--jp-code-line-height);
-  font-size: var(--jp-code-font-size);
```

Comment:
as mentioned, leaving this managed _by default_ by the theme would be nicer for theme implementers...